### PR TITLE
switch to numba 0.48 

### PR DESCRIPTION
**Pull request by Hardcode84**
latest commit in release0.48 branch

### CODE REVIEW

**Comment by PokhodenkoSA** [comment]
@Hardcode84 no problems after rebase? Cool!
How far this Numba commit from the current Numba master?

**Comment by PokhodenkoSA** [comment]
I see. It is exactly 0.48.0 tag. So we could update README with recommendation to install numba=0.48 and possibly use Python=3.8. @Hardcode84 is it correct?

**Comment by Hardcode84** [comment]
Didn't tried with python 3.8, do we have any CI pipelines with it?

**Comment by PokhodenkoSA** [comment]
It is upgrade b/c `ef119bcd` is before `cc7e7c7c`.

**Comment by Vyacheslav-Smirnov** [comment]
Actually IDP does not have Python 3.8 - so suppose SDC should have recommendation to use Python 3.7

File: buildscripts/numba-conda-recipe/recipe/meta.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ package:
 
 source:
   git_url: https://github.com/numba/numba
-  git_tag: ef119bcd1733ff49d71bdf2da8a66e91bb704f83
+  git_tag: cc7e7c7cfa6389b54d3b5c2c95751c97eb531a96
```
Comment (Vyacheslav-Smirnov) [inline buildscripts/numba-conda-recipe/recipe/meta.yaml]:
```suggestion
  git_tag: 0.48.0
```

Comment (Vyacheslav-Smirnov) [reply buildscripts/numba-conda-recipe/recipe/meta.yaml]: Suggest to specify here 0.48.0 tag

**Comment by Vyacheslav-Smirnov** [comment]
@Hardcode84 , could you please update your branch to the latest changes in master?
There are changes in Readme.rst which should be updated

**Pull request merged**